Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "multiset"
-version = "0.0.5"
+version = "0.1.0"
```

Comment:
As far as I understand semver we can introduce breaking changes all the time as long as we're in `0.0.x`.

Reply:
Right. Wouldn't it be reasonable at this point to imagine that we could want to release non-breaking changes though? In order to open that door, we could still release it as 0.1.0. I beleive this is the reason why packages are always created at 0.1.0 by default.

Reply:
To me this library seems very undecided on many major things and there don't seem to be any heavy users yes but either ways I wouldn't mind. What do you think @jmitchell?

Reply:
Please open a separate issue for this discussion and revert.